project_task_category/migrations/10.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2018 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    cr = env.cr
+    if openupgrade.table_exists(cr, 'project_category_main') and \
```

Comment:
Do the contrary check and return for saving one indentation level on the rest of the code.

Reply:
done

Reply:
I still don't see it

Reply:
Sorry. Incorrect comprehension.  Now you can have a look.